test/bigtest/tests/forgotUsername-test.js

```diff
@@ -0,0 +1,190 @@
+import { expect } from 'chai';
+import translations from '../../../translations/stripes-core/en';
+import setupApplication from '../helpers/setup-application';
+import ForgotUsernameInteractor from '../interactors/ForgotUsername';
+
+describe('Forgot username form test', () => {
+  setupApplication({ disableAuth: false });
+
+  beforeEach(function () {
+    this.visit('/forgot-username');
+  });
+
+  const forgotUsernamePage = new ForgotUsernameInteractor();
+  const {
+    inputField,
+    submitButton,
+    submitButton: { button },
+    mainHeading,
+    callToActionParagraph,
+    errorsWrapper,
+    errorsWrapper: { errorsContainer },
+    statusPage,
+    statusPage:
+        {
+          heading: statusPageHeading,
+          notificationParagraph,
+          cautionParagraph,
+        },
+  } = forgotUsernamePage;
+  const invalidInput = 'asdfgh12345';
+  const nonExistingRecord = '12345';
+  const existingRecord = '127-699-8925';
+
+  describe('Forgot form text input field tests', () => {
+    it('Should display a field to enter the email or phone number', () => {
+      expect(inputField.isPresent).to.be.true;
+    });
+
+    it('Should have an empty value', () => {
+      expect(inputField.val).to.equal('');
+    });
+  });
+
+  describe('Forgot form submit button tests', () => {
+    it('Should display a "Continue" button to submit a request', () => {
+      expect(button.isPresent).to.be.true;
+    });
+
+    it('Should have a disabled "Continue" button by default', () => {
+      expect(submitButton.isDisabled).to.be.true;
+    });
+  });
+
+  describe('Forgot form submit button test after filling the input', () => {
+    beforeEach(async () => {
+      await inputField.fillInput(invalidInput);
+    });
+    it('Should have an enabled submit button', () => {
+      expect(submitButton.isDisabled).to.be.false;
+    });
+  });
+
+  describe('Forgot form headings tests', () => {
+    it('Should display the heading', () => {
+      expect(mainHeading.isPresent).to.be.true;
+    });
+
+    it(`Should have the main heading content equal to forgot username label 
+    in english translation`, () => {
+      expect(mainHeading.text).to.equal(translations['label.forgotUsername']);
+    });
+
+    it('Should display the paragraph', () => {
+      expect(callToActionParagraph.isPresent).to.be.true;
+    });
+
+    it(`Should have the paragraph content equal to forgot username or password
+    call to action label in english translation`, () => {
+      expect(callToActionParagraph.text).to.equal(
+        translations['label.forgotUsernameOrPasswordCallToAction']
+      );
+    });
+  });
+
+  describe('Error container initial behaviour', () => {
+    describe('Forgot form error container integration test', () => {
+      it('Should display the error container wrapper', () => {
+        expect(errorsWrapper.isPresent).to.be.true;
+      });
+
+      it('Should not display the error container', () => {
```

Comment:
I think it should be `it.always`. Read about it here [here](https://github.com/bigtestjs/convergence). Check other similar cases.